main.go

```diff
@@ -32,18 +33,20 @@ type config struct {
 }
 
 const (
-	maxCommitBodyLenght = 72
+	trimEnding = "..."
 )
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
+var maxCommitMessageLength int
```

Comment:
Do not use global variable if you can.
The previous `maxCommitBodyLenght` was global because it had a default value, and it was easier to modify it.
But because the `maxCommitMessageLength` is a variable with no default value I would recommend to use a local variable instead and pass it between the functions.

Reply:
If I do not use a global variable, then I have to pass it to printLogAndExportEnv every time that method is called, but it is only needed in only 2 cases.

Other way is to make it a getter, and call it when needed, but it would be still redundant.

Reply:
i agree with @BirmacherAkos :
- modify `initCommitMessageLength`: it should just read the envman config and return the maxLenght
- printLogAndExportEnv is only called in the mainE, where you call the current init initCommitMessageLength